### DISCUSSION TITLE
Are there any good proxies available?

### DISCUSSION THREAD

**Discussion by 7trail** [acheong08/ChatGPT#1062] [General]
Apologies if this is (slightly) redundant. I've been looking at the issues and discussion pages and have noticed a consistent trend that the API has gotten slower over the last three days. I can attest to this same issue. From what I know, this was caused by a proxy receiving a DDoS attack. Are there any other good proxies available for use?

**Comment by acheong08** [comment]
I just added some measures on Cloudflare to stop usage outside of this library to prevent abuse. Try upgrading

**Comment by 7trail** [comment]
The new version is not on pip/pypi yet. Should I upgrade from the git repo?

**Comment by 7trail** [comment]
Currently trying to figure out how to upgrade. I am on 3.1.4 and it is still not working. What am I doing wrong? Apologies.

**Reply by acheong08** [reply]
Forgot to publish release https://github.com/acheong08/ChatGPT/releases/tag/3.1.5
